source/dub/dub.d

```diff
@@ -80,7 +86,7 @@ PackageSupplier[] defaultPackageSuppliers()
 	return [
 		new FallbackPackageSupplier(
 			new RegistryPackageSupplier(URL(defaultRegistryURL)),
-			new RegistryPackageSupplier(URL(fallbackRegistryURL))
+			fallbackRegistryURLs.map!(x => cast(PackageSupplier) new RegistryPackageSupplier(URL(x))).array
```

Comment:
Could have stayed with the recursive structure.
```d
fallbackRegistryURLs.reduce!((agg, url) => new FallbackPackageSupplier(agg, new RegistryPackageSupplier(URL(x))))
```

Reply:
Hmm, I thought the "linear" way is more intuitive, but I don't really mind. If you feel strongly the recursive structure, feel free to change it back.